lib/pharos/addon.rb

```diff
@@ -11,18 +11,15 @@ module Pharos
   # @param name [String]
   # @return [Pharos::Addon]
   def self.addon(name, &block)
-    klass = Class.new(Pharos::Addon, &block).tap do |addon|
+    Class.new(Pharos::Addon, &block).tap do |addon|
       addon.addon_location = File.dirname(block.source_location.first)
```

Comment:
`block.source_location` is empty which makes this PR useless (no resources are actually applied because addon cannot find any yamls). E2E tests did show this ...